src/Symfony/Component/Translation/Bridge/Loco/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/loco-translation-bridge",
+    "type": "symfony-bridge",
+    "description": "Symfony Loco Translation Bridge",
+    "keywords": ["loco", "translation", "provider"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Mathieu Santostefano",
+            "homepage": "https://github.com/welcomattic"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/translation": "^5.2"
+    },
+    "require-dev": {
+        "symfony/http-client": "^5.2"
+    },
```

Comment:
We need to add `symfony/http-client-contracts` or move `symfony/http-client` up to "require".

I prefer the former.